toolchain/sem_ir/value_stores.h

```diff
@@ -189,18 +189,24 @@ class TypeStore : public ValueStore<TypeId> {
   explicit TypeStore(InstStore* insts) : insts_(insts) {}
 
   // Returns the ID of the instruction used to define the specified type.
```

Comment:
Should this say "constant" instead of "instruction"?
```suggestion
  // Returns the ID of the constant used to define the specified type.
```

Reply:
Done.